pkg/kubelet/server/auth_test.go

```diff
@@ -82,60 +88,76 @@ const (
 type AuthzTestCase struct {
 	Method, Path string
 
-	ExpectedVerb, ExpectedSubresource string
+	ExpectedVerb         string
+	ExpectedSubresources []string
 }
 
-func (a *AuthzTestCase) AssertAttributes(t *testing.T, attrs authorizer.Attributes) {
-	expectedAttributes := authorizer.AttributesRecord{
-		User:            AuthzTestUser(),
-		APIGroup:        "",
-		APIVersion:      "v1",
-		Verb:            a.ExpectedVerb,
-		Resource:        "nodes",
-		Name:            authzTestNodeName,
-		Subresource:     a.ExpectedSubresource,
-		ResourceRequest: true,
-		Path:            a.Path,
+func (a *AuthzTestCase) AssertAttributes(t *testing.T, attrs []authorizer.Attributes) {
+
+	var expectedAttributes []authorizer.AttributesRecord
+	for _, subresource := range a.ExpectedSubresources {
+		expectedAttributes = append(expectedAttributes, authorizer.AttributesRecord{
+			User:            AuthzTestUser(),
+			APIGroup:        "",
+			APIVersion:      "v1",
+			Verb:            a.ExpectedVerb,
+			Resource:        "nodes",
+			Name:            authzTestNodeName,
+			Subresource:     subresource,
+			ResourceRequest: true,
+			Path:            a.Path,
+		})
 	}
 
-	assert.Equal(t, expectedAttributes, attrs)
+	assert.ElementsMatch(t, expectedAttributes, attrs)
```

Comment:
`ElementsMatch` ignores order, but I think order is significant here. Can this just stick with `Equal`?

Reply:
Good call. Done!